components/example-notebook-servers/README.md

```diff
@@ -1,113 +1,130 @@
 # Example Notebook Servers
 
-> 🛑️️ Images are provided as __examples__ and are supported on a best-effort basis.
+> These images are provided as __examples__, and are supported on a best-effort basis.
+> <br>
 > Contributions are greatly appreciated.
 
 ## Images
 
-### Images // Base
+This chart shows how the images are related to each other (the nodes are clickable links to the Dockerfiles):
 
-These images provide a common starting point for Kubeflow Notebook containers.
-See the [custom images guide](#custom-images) to learn how to extend them with your own packages.
+```mermaid
+graph TD
+  Base[<a href='https://github.com/thesuperzapper/kubeflow/tree/master/components/example-notebook-servers/base'>Base</a>] --> Jupyter[<a href='https://github.com/thesuperzapper/kubeflow/tree/master/components/example-notebook-servers/jupyter'>Jupyter</a>]
```

Comment:
I'd be better if we point to the kubeflow/kubeflow repository here.

Reply:
Ah, good catch, will quickly fix that.

Reply:
Fixed in https://github.com/kubeflow/kubeflow/pull/7357/commits/e38abb7ce3d47260df6d2980cd22e4b3a6d2b825